easybuild/easyconfigs/b/biogeme/biogeme-3.1.2-foss-2021a-Python-3.9.5.eb

```diff
@@ -0,0 +1,36 @@
+easyblock = 'PythonBundle'
+
+name = 'biogeme'
+version = '3.1.2'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://pypi.python.org/pypi/biogeme'
+description = """Biogeme is a open source Python package designed for
+the maximum likelihood estimation of parametric models in general,
+with a special emphasis on discrete choice models."""
+
+use_pip = True
+sanity_pip_check = True
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+
+dependencies = [
+    ('Python', '3.9.5'),
+    ('SciPy-bundle', '2021.05')
+]
+
+exts_list = [
+    ('Unidecode', '1.2.0', {
+        'checksums': ['8d73a97d387a956922344f6b74243c2c6771594659778744b2dbdaad8f6b727d'],
+    }),
+    (name, version, {
+        'checksums': ['28aecfa3701c293e3b3c6bcab68322d77d1721e10b18d987f592651333f1e438'],
+    }),
+]
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['lib/python%(pyshortver)s/site-packages/biogeme'],
+}
+
+moduleclass = 'math'
```

Comment:
Please add a newline at the end of the file.

Reply:
@verdurin newline added, thanks for pointing this out.